network_importer/adapters/netbox_api/adapter.py

```diff
@@ -117,6 +120,7 @@ def load(self):
             if nb_device["primary_ip"]:
                 device.primary_ip = nb_device["primary_ip"].get("address")
 
+            device = apply_model_flag(device, nb_device)
```

Comment:
I think we should move `apply_model_flag` and `_is_tag_present` under NetBoxAPIAdapter (probably as classmethod) this way if someone want to update the logic without reimplementing everything it's possible

Reply:
Updated.